wire-library/wire-runtime-swift/src/test/swift/CodableTests.swift

```diff
@@ -29,10 +29,10 @@ final class CodableTests: XCTestCase {
           "repeated_int32":[1,2,3],
           "opt_uint64":"4",
           "opt_double":6,
-          "map_int32_string":[
```

Comment:
I'd be shocked if anyone actually depended upon this Codable representation since it's not remotely idiomatic JSON